_posts/2018-04-17-changes-to-the-api.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  Important Changes to the OpenDota API
+date: 2018-04-17
+summary: Introducing the Free and Premium Tiers
+---
+**IMPORTANT: Users making more than 50,000 API calls per month will need to sign up for an API key in order to keep using the API.**
```

Comment:
"in order to enable usage beyond this limit"?

Reply:
changed